source/_integrations/statistics.markdown

```diff
@@ -12,10 +12,9 @@ ha_codeowners:
 ha_domain: statistics
 ---
 
-The `statistics` sensor platform consumes the state from other sensors. It exports the `mean` value as state and the following values as attributes: `count`, `mean`, `median`, `stdev`, `variance`, `total`, `min_value`, `max_value`, `min_age`, `max_age`, `change`, `average_change` and `change_rate`. If it's a binary sensor then only state changes are counted.
+The `statistics` sensor platform consumes the state from other sensors. It exports the `mean` value as state and the following values as attributes: `count`, `mean`, `median`, `stdev`, `variance`, `total`, `min_value`, `max_value`, `min_age`, `max_age`, `change`, `average_change` and `change_rate`. If the source is a binary sensor then only state changes are counted.
 
-If you are running the [recorder](/integrations/recorder/) component, on startup the data is read from the database. So after a restart of the platform, you will immediately have data available. If you're using the [history](/integrations/history/) component, this will automatically also start the `recorder` integration on startup.
-If you are *not* running the `recorder` component, it can take time till the sensor starts to work because a couple of attributes need more than one value to do the calculation.
+Assuming the [recorder](/integrations/recorder/) and [history](/integrations/history/) components are running (either configured explicitly or via [`default_config:`](https://www.home-assistant.io/integrations/default_config/)), historical sensor data is read from the database on startup and is available immediately after a restart of the platform. Without these components, it can take time for the sensor to start reporting data because some attribute calculations require more than one value. 
```

Comment:
- Use relative links from the root of the website
- Removed space at end of paragraph
- `components` is not a term we use anymore and should be removed.

```suggestion
Assuming the [recorder](/integrations/recorder/) and [history](/integrations/history/) integrations are running (either configured explicitly or via [`default_config:`](/integrations/default_config/)), historical sensor data is read from the database on startup and is available immediately after a restart of the platform. Without these integrations, it can take time for the sensor to start reporting data because some attribute calculations require more than one value.
```

Lastly, this now renders differently. Where the older text stated you need either recorder or the history (which automatically activates recorder), it now states you need both?